reactioncommerce/kinetic
# [MCOSS-14] Allow users to see their shops and information after logging in

Signed-off-by: Chloe <pinkcloudvnn@gmail.com>
Jira ticket: https://jira.mailchimp.com/browse/MCOSS-14

Testing instructions:
- Start the app
- Login into a registered account
- Expect to be able to see the account profile toolbar on the sidebar
<img width="455" alt="image" src="https://user-images.githubusercontent.com/33818318/173798449-84dad5f1-e2b0-44ed-a6d5-ab36b65610e8.png">
